README.md

```diff
@@ -67,7 +67,7 @@ of details about the execution history.
 
 ## Install Temporal CLI
 
-[Command Line Interface Documentation](https://docs.temporal.io/docs/08_cli)
+[Command Line Interface Documentation](https://docs.temporal.io/docs/08_running_temporal/02_cli)
```

Comment:
verified that this is a real page: https://docs.temporal.io/docs/08_running_temporal/02_cli

![image](https://user-images.githubusercontent.com/218696/77216066-f8eb4280-6ad4-11ea-8894-1a979a4b1cae.png)